### ISSUE TITLE
Incorrect order currency 

### ISSUE THREAD

**Issue by moondog604** [woothemes/woocommerce#34407]
### Prerequisites

- [X] I have carried out troubleshooting steps and I believe I have found a bug.
- [X] I have searched for similar bugs in both open and closed issues and cannot find a duplicate.

### Describe the bug

We have been having issues where customers are randomly being charged the incorrect currency during checkout.

I found the issue and have been able to repeatedly trigger it. We are using the Aelia Currency Switcher.

I deleted all of the unused plugins and disabled all of active ones with the exception of the plugins directly associated with the checkout process. I updated all of the associated addons to the latest version. Nothing corrected this.

### Expected behavior

Checkout to always charge the currency based on the customers shipping country

### Actual behavior

The checkout process doesn't appear to honor the shipping country of the billing address when you uncheck "Ship to a different address?" if the country was visibly different. If you change it to the correct country before you uncheck "Ship to a different address?" it updates the shipping country with the correct country.

### Checkout

**Step 1** If you go in and have different a different country select for the shipping address initially

![checkout-a](https://user-images.githubusercontent.com/22012926/185777725-7be980dd-4ede-4bbc-950e-4966dcbebbf5.png)

**Step 2** Then uncheck "Ship to a different address?" it fails to update the country when the order is saved.

![checkout-b](https://user-images.githubusercontent.com/22012926/185777729-45c5fc0e-00c2-4fac-9af6-c8e1cdb08f0f.png)

### Order

You can see in the screen shot the province is BC and there are Canadian Taxes on the order, but the currency is shown as USD. It does the same thing if you do step 1 and 2 with the countries flipped. Once the payments go to stripe it charges the wrong currency when this bug occurs.

![order](https://user-images.githubusercontent.com/22012926/185777981-b64388e2-305b-4463-9c3d-492a1267709e.png)

### WordPress Environment

`
### WordPress Environment ###

WordPress address (URL): https://solusdecorstg.wpengine.com
Site address (URL): https://solusdecorstg.wpengine.com
WC Version: 6.8.0
REST API Version: ✔ 6.8.0
WC Blocks Version: ✔ 8.0.0
Action Scheduler Version: ✔ 3.4.0
Log Directory Writable: ✔
WP Version: ❌ 6.0 - There is a newer version of WordPress available (6.0.1)
WP Multisite: –
WP Memory Limit: 512 MB
WP Debug Mode: –
WP Cron: ✔
Language: en_CA
External object cache: ✔

### Server Environment ###

Server Info: Apache
PHP Version: 8.0.21
PHP Post Max Size: 100 MB
PHP Time Limit: 3600
PHP Max Input Vars: 10000
cURL Version: 7.74.0
OpenSSL/1.1.1n

SUHOSIN Installed: –
MySQL Version: 5.7.38-41-log
Max Upload Size: 100 MB
Default Timezone is UTC: ✔
fsockopen/cURL: ✔
SoapClient: ✔
DOMDocument: ✔
GZip: ✔
Multibyte String: ✔
Remote Post: ✔
Remote Get: ✔

### Database ###

WC Database Version: 6.8.0
WC Database Prefix: wp_
Total Database Size: 111.92MB
Database Data Size: 64.18MB
Database Index Size: 47.74MB
wp_woocommerce_sessions: Data: 0.05MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_api_keys: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_attribute_taxonomies: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_downloadable_product_permissions: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_woocommerce_order_items: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_order_itemmeta: Data: 0.09MB + Index: 0.09MB + Engine InnoDB
wp_woocommerce_tax_rates: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_woocommerce_tax_rate_locations: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_shipping_zones: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_woocommerce_shipping_zone_locations: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_shipping_zone_methods: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_woocommerce_payment_tokens: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_payment_tokenmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_log: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_actionscheduler_actions: Data: 0.08MB + Index: 0.13MB + Engine InnoDB
wp_actionscheduler_claims: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_actionscheduler_groups: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_actionscheduler_logs: Data: 0.05MB + Index: 0.03MB + Engine InnoDB
wp_aelia_dismissed_messages: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_aioseo_notifications: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_aioseo_posts: Data: 1.52MB + Index: 0.09MB + Engine InnoDB
wp_ak_twitter: Data: 0.16MB + Index: 0.05MB + Engine InnoDB
wp_bpspro_login_security: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_bpspro_seclog_ignore: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_cbnetpo_ping_optimizer: Data: 1.52MB + Index: 0.00MB + Engine InnoDB
wp_commentmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_comments: Data: 0.05MB + Index: 0.09MB + Engine InnoDB
wp_e_events: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_e_notes: Data: 0.02MB + Index: 0.17MB + Engine InnoDB
wp_e_notes_users_relations: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_e_submissions: Data: 0.02MB + Index: 0.27MB + Engine InnoDB
wp_e_submissions_actions_log: Data: 0.02MB + Index: 0.11MB + Engine InnoDB
wp_e_submissions_values: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_fo_elements: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_fo_usable_fonts: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_gf_draft_submissions: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_gf_entry: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_gf_entry_meta: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_gf_entry_notes: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_gf_form: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_gf_form_meta: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_gf_form_revisions: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_gf_form_view: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_imagify_files: Data: 0.02MB + Index: 0.08MB + Engine InnoDB
wp_imagify_folders: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_iwp_backup_status: Data: 0.09MB + Index: 0.00MB + Engine InnoDB
wp_iwp_file_list: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_iwp_processed_iterator: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_links: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_ms_snippets: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_nf3_actions: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_nf3_action_meta: Data: 0.11MB + Index: 0.00MB + Engine InnoDB
wp_nf3_chunks: Data: 0.16MB + Index: 0.00MB + Engine InnoDB
wp_nf3_fields: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_nf3_field_meta: Data: 0.31MB + Index: 0.00MB + Engine InnoDB
wp_nf3_forms: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_nf3_form_meta: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_nf3_objects: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_nf3_object_meta: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_nf3_relationships: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_nf3_upgrades: Data: 0.20MB + Index: 0.00MB + Engine InnoDB
wp_options: Data: 3.52MB + Index: 0.08MB + Engine InnoDB
wp_pmxe_exports: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_pmxe_google_cats: Data: 0.39MB + Index: 0.00MB + Engine InnoDB
wp_pmxe_posts: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_pmxe_templates: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_pmxi_files: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_pmxi_history: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_pmxi_images: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_pmxi_imports: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_pmxi_posts: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_pmxi_templates: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_popularpostsdata: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_popularpostsdatacache: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_popularpostssummary: Data: 3.52MB + Index: 5.55MB + Engine InnoDB
wp_postmeta: Data: 39.56MB + Index: 33.11MB + Engine InnoDB
wp_postrank: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_posts: Data: 4.52MB + Index: 1.33MB + Engine InnoDB
wp_py_jobs: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_py_jobs_data: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_redirection_404: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_redirection_groups: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_redirection_items: Data: 0.02MB + Index: 0.08MB + Engine InnoDB
wp_redirection_logs: Data: 0.02MB + Index: 0.08MB + Engine InnoDB
wp_seopressor_automatic_internal_links: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_seopressor_external_cloacked_links: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_seopressor_logs: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_seopressor_lsi: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_seopressor_roles_capabilities: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_seopressor_users_custom_roles: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_seopressor_visits: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_shortpixel_folders: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_shortpixel_meta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_shortpixel_postmeta: Data: 0.06MB + Index: 0.08MB + Engine InnoDB
wp_shortpixel_queue: Data: 0.02MB + Index: 0.09MB + Engine InnoDB
wp_smush_dir_images: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_snippets: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_termmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_terms: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_term_relationships: Data: 0.17MB + Index: 0.19MB + Engine InnoDB
wp_term_taxonomy: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_usermeta: Data: 0.39MB + Index: 0.09MB + Engine InnoDB
wp_users: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_vxc_sales_accounts: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_vxc_sales_log: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_admin_notes: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wc_admin_note_actions: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_category_lookup: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wc_customer_lookup: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_download_log: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_order_coupon_lookup: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_order_product_lookup: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_wc_order_stats: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_wc_order_tax_lookup: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_product_attributes_lookup: Data: 0.09MB + Index: 0.08MB + Engine InnoDB
wp_wc_product_download_directories: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_product_meta_lookup: Data: 0.14MB + Index: 0.30MB + Engine InnoDB
wp_wc_rate_limits: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_reserved_stock: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wc_tax_rate_classes: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_webhooks: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wfBadLeechers: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wfBlockedIPLog: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wfBlocks: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wfBlocksAdv: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wfConfig: Data: 0.06MB + Index: 0.00MB + Engine InnoDB
wp_wfCrawlers: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wfFileMods: Data: 1.52MB + Index: 0.00MB + Engine InnoDB
wp_wfHits: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wfIssues: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wfLeechers: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wfLockedOut: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wfLocs: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wfLogins: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wfReverseCache: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wfScanners: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wfStatus: Data: 0.13MB + Index: 0.09MB + Engine InnoDB
wp_wfThrottleLog: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wpr_rucss_resources: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_wpr_rucss_used_css: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_wt_iew_action_history: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wt_iew_cron: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wt_iew_ftp: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wt_iew_mapping_template: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_yarpp_related_cache: Data: 0.09MB + Index: 0.13MB + Engine InnoDB
wp_yoast_indexable: Data: 1.52MB + Index: 1.41MB + Engine InnoDB
wp_yoast_indexable_hierarchy: Data: 0.11MB + Index: 0.22MB + Engine InnoDB
wp_yoast_migrations: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_yoast_primary_term: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_yoast_seo_links: Data: 1.52MB + Index: 1.91MB + Engine InnoDB
wp_yoast_seo_meta: Data: 0.08MB + Index: 0.00MB + Engine InnoDB

### Post Type Counts ###

acf-field: 2
acf-field-group: 2
attachment: 2773
custom_css: 1
elementor_cf_db: 65
elementor_library: 31
gp_elements: 3
iwp_log: 734
jet-menu: 11
nav_menu_item: 24
nf_sub: 2793
oembed_cache: 4
page: 30
post: 464
product: 56
product_variation: 994
shop_order: 33

### Security ###

Secure connection (HTTPS): ✔
Hide errors from visitors: ✔

### Active Plugins (30) ###

InfiniteWP - Client: by Revmakx – 1.9.8
Advanced Custom Fields: by Delicious Brains – 5.12.3
Advanced Post Queries: by Shabti Kaplan – 1.0.15
Akismet Anti-Spam: by Automattic – 5.0
Classic Editor: by WordPress Contributors – 1.6.2
Code Snippets: by Code Snippets Pro – 3.2.0
Elementor Pro: by Elementor.com – 3.7.2
Elementor: by Elementor.com – 3.6.6
Enable Media Replace: by ShortPixel – 3.6.3
GP Premium: by Tom Usborne – 2.1.2
JetElements For Elementor: by Crocoblock – 2.6.6
JetMenu: by Crocoblock – 2.1.7
LoginPress - Hide Login: by WPBrigade – 1.2.3
LoginPress Pro: by WPBrigade – 2.5.2
LoginPress - Customizing the WordPress Login Screen.: by WPBrigade – 1.6.2
Ninja Forms - Salesforce CRM: by Stuart Sequeira – 3.3.1
Ninja Forms: by Saturday Drive – 3.6.12
Responsive Lightbox & Gallery: by dFactory – 2.4.1
Elementor Contact Form DB: by Sean Barton - Tortoise IT – 1.7
ShortPixel Image Optimizer: by ShortPixel – 5.0.8
SVG Support: by Benbodhi – 2.4.2
Advanced Editor Tools (previously TinyMCE Advanced): by Automattic – 5.6.0
Aelia Foundation Classes for WooCommerce: by Aelia – 2.3.0.220730
Payment Plugins for Stripe WooCommerce: by Payment Plugins
support@paymentplugins.com – 3.3.25

Variation Swatches for WooCommerce: by Emran Ahmed – 2.0.8
Aelia Currency Switcher for WooCommerce: by Aelia – 4.14.1.220803
WooCommerce: by Automattic – 6.8.1
WordPress Importer: by wordpressdotorg – 0.7
Yoast SEO: by Team Yoast – 19.5.1
WP Rocket: by WP Media – 3.11.3

### Dropin Plugins (2) ###

advanced-cache.php: advanced-cache.php
object-cache.php: Memcached Redux

### Must Use Plugins (7) ###

Force Strong Passwords - WPE Edition: by Jason Cosper – 1.8.0
InfiniteWP - Client Loader: by Revmakx – 1.0.1
WPE ElasticPress Autosuggest Logger: by WP Engine – 1.0.0
WP Engine Cache Plugin: by WP Engine – 1.0.10
WP Engine Seamless Login Plugin: by WP Engine – 1.6.0
WP Engine Security Auditor: by wpengine – 1.0.10
WP Engine System: by WP Engine – 5.0.1

### Settings ###

API Enabled: –
Force SSL: –
Currency: CAD ($)
Currency Position: left
Thousand Separator: ,
Decimal Separator: .
Number of Decimals: 2
Taxonomies: Product Types: external (external)
grouped (grouped)
simple (simple)
variable (variable)

Taxonomies: Product Visibility: exclude-from-catalog (exclude-from-catalog)
exclude-from-search (exclude-from-search)
featured (featured)
outofstock (outofstock)
rated-1 (rated-1)
rated-2 (rated-2)
rated-3 (rated-3)
rated-4 (rated-4)
rated-5 (rated-5)

Connected to WooCommerce.com: –
Enforce Approved Product Download Directories: –

### WC Pages ###

Shop base: #20923 - /shop-online/
Cart: #182 - /cart/
Checkout: #183 - /checkout/
My account: ❌ Page not set
Terms and conditions: ❌ Page not set

### Theme ###

Name: GeneratePress
Version: 3.1.3
Author URL: https://tomusborne.com
Child Theme: ❌ – If you are modifying WooCommerce on a parent theme that you did not build personally we recommend using a child theme. See: How to create a child theme
WooCommerce Support: ✔

### Templates ###

Overrides: –

### Admin ###

Enabled Features: activity-panels
analytics
coupons
customer-effort-score-tracks
experimental-products-task
experimental-import-products-task
experimental-fashion-sample-products
experimental-product-tour
shipping-smart-defaults
shipping-setting-tour
homescreen
marketing
mobile-app-banner
navigation
onboarding
onboarding-tasks
remote-inbox-notifications
remote-free-extensions
payment-gateway-suggestions
shipping-label-banner
subscriptions
store-alerts
transient-notices
wc-pay-promotion
wc-pay-welcome-page
wc-pay-subscriptions-page

Disabled Features: minified-js
settings

Daily Cron: ✔ Next scheduled: 2022-08-21 16:19:02 -07:00
Options: ✔
Notes: 7
Onboarding: completed

### Action Scheduler ###

Complete: 105
Oldest: 2022-07-20 23:14:06 -0700
Newest: 2022-08-20 22:37:29 -0700

Failed: 1
Oldest: 2022-03-11 10:06:52 -0800
Newest: 2022-03-11 10:06:52 -0800

Pending: 2
Oldest: 2022-08-21 04:07:14 -0700
Newest: 2022-08-21 04:07:14 -0700

### Status report information ###

Generated at: 2022-08-20 23:07:20 -07:00
`

### Isolating the problem

- [X] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [ ] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/storefront/).
- [X] I can reproduce this bug consistently using the steps above.

**Comment by moondog604** [comment]
I went through their orders on the production version of the website since at least the beginning of June when they launched.

I also looked at the browser console to see if there was an error occurring when the checkbox for "Ship to a different address?" was toggled.  As far as I can tell it's failing to use the billing country when you uncheck it.

**Comment by jonathansadowski** [comment]
Hi @moondog604,

Thanks for the report. You mentioned that you disabled most plugins except for those required for checkout... Is there any series of steps you're able to take to reproduce an issue with just WooCommerce enabled?

It sounds like there's a possibility that there may be an issue with Aelia Currency Switcher, and that you may need to reach out to them to report that. If there's a series of steps you can give us to reproduce an issue without that or other plugins active, please let us know and we'll try to reproduce.

**Comment by moondog604** [comment]
Hello @jonathansadowski 

At this point I am 99.9% sure you are correct. I don't think the currency switcher they have installed has the logic to handle this so late in the checkout process.

I most likely will be closing this in the next few days.

Thank you for the response!

**Comment by moondog604** [comment]
Hello again @jonathansadowski

Thank you so much for taking the time to get back to me.

They fixed the plugin for us, and I am just waiting for official patch to drop to install.

Ironically I had a similar issue in a shopping cart I built. I wasn't aware the checkout form was still submitting the values of the shipping fields, and I guess I didn't test enough address combinations to see the issue occurring. I ended up changing my "ship to billing" checkbox to replace the shipping address with the billing address in the hidden shipping fields.
